python_modules/libraries/dagster-k8s/dagster_k8s_tests/unit_tests/test_launcher.py

```diff
@@ -202,7 +202,10 @@ def test_launcher_with_k8s_config(kubeconfig_file):
     container_context_config = {
         "k8s": {
             "run_k8s_config": {
-                "container_config": {"command": ["echo", "REPLACED"]},
+                "container_config": {
+                    "command": ["echo", "REPLACED"],
+                    "args": ["&&", "shutdown.sh"],
```

Comment:
considered `additional_args` or `suffix_args` but these config sections are designed to mirror the k8s properties they set directly. Maybe its fine to have an non-k8s entry? Could also consider `additional_args` passed in elsewhere in the config structure